### DISCUSSION TITLE
Searching - Can only select 999 items

### DISCUSSION THREAD

**Discussion by ghost** [photoprism/photoprism#2518] [Q&A]
Trying to add photos to a new album.  For this example, I searched "state:Georgia".  It returned over 1200 results but I'm only allowed to select 999.
Is there a workaround to add all of the items included in the search into an album?  The only workaround I found was to sort the results by name or whatever, select the first 999 and add...then select the next ##'s.  It's just hard to keep track that way. If I start from the bottom and work my way up, if I selected more than 999 it gives me a message and then selects the FIRST 999 from the top down. Seems backwards, it should keep the one's I initially selected. At least that's the experience I was expecting.

Also another workaround, after adding the images to the new album, I can verify if I missed any by searching, in this example, "state:georgia unsorted:yes"

**Comment by lastzero** [comment]
First of all, there should already be an automatically created and updated album for the state in Places > States. You can clone its contents without selecting individual pictures if you don't want new pictures to appear automatically. Once we've built a proper UI for it, you can also create dynamic albums based on search filters on your own. Adding thousands of items instead of using a filter isn't very efficient. That's why there is a limit.

**Reply by graciousgrey** [reply]
Here is the related issue for conditional albums: https://github.com/photoprism/photoprism/issues/1431
